nengo_gui/examples/tutorial/20-spa-actions.py

```diff
@@ -20,18 +22,19 @@
 #
 # The following code defines four actions.  The utilities look like
 # this: "dot(vision, DOG)".  This means that the utility of the action
-# is the dot product of whatever vector is in the vision Buffer and
+# is the dot product of whatever vector is in the vision State and
 # the ideal vector for DOG.  This will be a value near 1 if you see a
-# dog, and will be near zero otherwise.  The closer what you see is to 
+# dog, and will be near zero otherwise.  The closer what you see is to
 # a dog, the higher the utility.  The effect of the action is simply
-# to set the speech Buffer to the vector for BARK ("speech=BARK").
+# to set the speech State to the vector for BARK ("speech=BARK").
 #
 # Try running the model.  Use "Set value" on the vision graph to set the
 # input to DOG.  The speech output should become BARK.  Try other inputs,
 # such as CAT or COW.  You can also try a combination, such as
 # RAT+0.5*COW.  The system will (almost always) select a single action
 # to perform, and that will be the action with the highest utility at that
-# moment.
+# moment. If there is no input, or the input contains just noise, the speech
+# State is set to zero ("speech=0").
```

Comment:
I'm not sure about the "noise" comment.  How about "If there is no input, or the input is not sufficiently similar
to any of the four animals it knows, then the speech State is set to zero ("speech=0")"